scripts/update_configs_test.py

```diff
@@ -0,0 +1,308 @@
+# coding: utf-8
+#
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for scripts/update_configs.py."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import getpass
+import os
+import sys
+import tempfile
+
+from core.tests import test_utils
+import python_utils
+
+from . import common
+from . import update_configs
+
+_PARENT_DIR = os.path.abspath(os.path.join(os.getcwd(), os.pardir))
+_PY_GITHUB_PATH = os.path.join(_PARENT_DIR, 'oppia_tools', 'PyGithub-1.43.7')
+sys.path.insert(0, _PY_GITHUB_PATH)
+
+# pylint: disable=wrong-import-position
+import github # isort:skip
+# pylint: enable=wrong-import-position
+
+INVALID_FECONF_CONFIG_PATH = os.path.join(
+    os.getcwd(), 'core', 'tests', 'release_sources',
+    'invalid_feconf_updates.config')
+VALID_FECONF_CONFIG_PATH = os.path.join(
+    os.getcwd(), 'core', 'tests', 'release_sources',
+    'valid_feconf_updates.config')
+FECONF_CONFIG_PATH_WITH_EXTRA_LINE = os.path.join(
+    os.getcwd(), 'core', 'tests', 'release_sources',
+    'feconf_updates_with_extra_line.config')
+MOCK_LOCAL_FECONF_PATH = os.path.join(
+    os.getcwd(), 'core', 'tests', 'release_sources',
+    'feconf.txt')
+
+
+class UpdateConfigsTests(test_utils.GenericTestBase):
+    """Test the methods for updating configs."""
+    def setUp(self):
+        super(UpdateConfigsTests, self).setUp()
+        self.mock_repo = github.Repository.Repository(
+            requester='', headers='', attributes={}, completed='')
+        def mock_get_current_branch_name():
+            return 'release-1.2.3'
+        def mock_release_scripts_exist():
+            pass
+        def mock_get_organization(unused_self, unused_name):
+            return github.Organization.Organization(
+                requester='', headers='', attributes={}, completed='')
+        def mock_get_repo(unused_self, unused_org):
+            return self.mock_repo
+        def mock_open_tab(unused_url):
+            pass
+        # pylint: disable=unused-argument
```

Comment:
Use unused_ and get rid of the pragmas. Ditto below.

Reply:
Same case as in the reply [here](https://github.com/oppia/oppia/pull/7779#discussion_r337943556).